crowbar_framework/spec/models/api/upgrade_spec.rb

```diff
@@ -178,12 +178,20 @@
         with("state:crowbar_upgrade AND (roles:database-server OR roles:rabbitmq-server)").
         and_return([NodeObject.find_node_by_name("testing.crowbar.com")])
       )
+      allow_any_instance_of(NodeObject).to receive(:run_ssh_cmd).and_return(
+        stdout: "",
+        stderr: "",
+        exit_code: 1
+      )
+
```

Comment:
I'm not sure what consistency are we aiming for here, but to me it looks like you should have that empty line before the `expect` call and not between those 2 `allow`s